#Lab | Hypothesis Testing

**Instructions

It is assumed that the mean systolic blood pressure is μ = 120 mm Hg. In the Honolulu Heart Study, a sample of n = 100 people had an average systolic blood pressure of 130.1 mm Hg with a standard deviation of 21.21 mm Hg. Is the group significantly different (with respect to systolic blood pressure!) from the regular population?

-- Set up the hypothesis test.
-- Write down all the steps followed for setting up the test.
-- Calculate the test statistic by hand and also code it in Python. It should be 4.76190. 
-- What decision can you make based on this calculated value?


In [9]:
#Set up the hypothesis test. 

In [17]:
import numpy as np
import pandas as pd
import scipy.stats as stats
import matplotlib.pyplot as plt
import seaborn as sns

%matplotlib inline

In [27]:
population_mean = 120
sample_mean = 130.1
standard_deviation = 21.21 
sample_size = 100


#Problem statement

Null Hypothesis : The mean systolic blood pressure in the study group is equal to the population mean (μ=120).
Alternative Hypothesis : The mean systolic blood pressure in the study group is significantly different from the population mean (120μ).


In [28]:
z = ( sample_mean - population_mean ) / ( standard_deviation / np.sqrt(100) )
print("Our z score is: {:.2f}".format(z))

Our z score is: 4.76


In [29]:
zc = stats.norm.ppf(1-(0.05/2))
print("Our zc score is: {:.2f}".format(zc))

Our zc score is: 1.96


In [1]:
# We can say we reject the Null Hypothesis. 

** Comparing the z-score to the critical value: ∣4.76∣≫1.96

Conclusion:
Since the absolute value of the z-score is much greater than the critical value, the null hypothesis is rejected. This suggests that the mean systolic blood pressure in the study group is significantly different from the regular population.

